### ISSUE TITLE
Service mode

### ISSUE THREAD

**Issue by weggis** [LogicInteractive/KontentumClient#15]
Would it be possible to implement a new command eg. "service" which revives explorer.exe (and ideally quits the running application)?

**Comment by datee** [comment]
Yep, that should be unproblematic i think!

**Comment by datee** [comment]
But quitting the running app is not possible if its running from a bat file or something similar..

**Comment by weggis** [comment]
Yes - understandable :-) But quitting the registered .exe file from launchpath is doable?

**Comment by datee** [comment]
Yes, we used to have this function in an older build 😊

**Comment by datee** [comment]
Actually the function is still there, can be reenabled without the crash - management
